public/js/test/fixtures/backbone.pause.onEnter.json

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
Yeah. I'm not sure what maintaining this is going to look like. It might be possible to automate generating these fixtures though: setup a devtools connection and automatically go to the right place and dump the protocol response to these files. It seems useful to have them, so let's just see how it feels over time.